### PR TITLE
fixes to mirador, fixes for logout

### PR DESCRIPTION
**Pull request by dnoneill**

### CODE REVIEW

File: annonatate/static/mirador-index.js [hunk 1]
```diff
@@ -23,17 +23,27 @@ const annoview = Vue.component('annoview', {
         const loadedManifest = manifesturl ? manifesturl.toString() : manifests[0];
         var manifestload = manifests.map(ma => JSON.parse(`{ "manifestUri": "${ma}"}`));
         manifestload.push({"manifestUri": loadedManifest})
+        const check = loadedManifest.indexOf(this.originurl) > -1;
+        const layout = check ? "1x2" : "1x1";
+        windowObjs = [{
```
Comment (pbinkley) [inline annonatate/static/mirador-index.js:28]: So the change is to cause Mirador to open with two windows if I provide my own manifest uri, but only one if I'm using one of the pre-configured manifests? I don't get what this is for.

Comment (dnoneill) [reply annonatate/static/mirador-index.js:28]: Hi Peter,
So this is a really hacky way of getting around a problem I found with Mirador. If there is an annotation list loaded in the manifest and I load those same annotations in Mirador, they become not editable which is bad considering the point of the application. I found that in the annotation server, I can delete the annotation list from the manifest data, the problem I ran into is that it does not update that you removed that annotation when you load regularly. The only way I found to fix this was to load the same manifest twice, remove the first loaded manifest, and then the annotations become editable. It is not an ideal solution, but I spent multiple days on this and it was the only thing I could get to work. Let me know if you have any questions or recommendations.

Comment (pbinkley) [reply annonatate/static/mirador-index.js:28]: ok, that sounds good. The only thing I'd suggest is to put in an issue for Mirador, and put the link to the issue in a comment here, to make it easier to remove the hack if the problem gets fixed upstream.

Comment (dnoneill) [reply annonatate/static/mirador-index.js:28]: This is using Mirador 2 which I don't believe is being supported or further developed so I am going to hold off on adding an issue until I can figure out how to check to see if this same problem exists in Mirador 3.

**Comment by dnoneill** [comment]
@pbinkley Anymore feedback, I want to get this integrated before next week.

**Pull request merged**